templates/auth_gssapi.conf.erb

```diff
@@ -0,0 +1,14 @@
+
+<Location /users/extlogin>
+  SSLRequireSSL
+  AuthType GSSAPI
+  AuthName "GSSAPI Single Sign On Login"
+  GssapiCredStore keytab:<%= scope.lookupvar('::foreman::http_keytab') %>
+  GssapiSSLonly On
```

Comment:
this is not the default :roll_eyes: 

probably also want to set 
```
GssapiUseSessions …
GssapiAllowedMech krb5
```

Reply:
After thinking, `GssapiUseSessions` is not really much of a concern, as we only hit kerberos when the user visits `/extlogin`, not on *every* page load.